plugin.video.youtube/README.md

```diff
@@ -1,14 +1,8 @@
-![](https://raw.githubusercontent.com/bromix/repository.bromix.storage/master/plugin.video.youtube/icon.png)
+![](https://raw.githubusercontent.com/Kolifanes/plugin.video.youtube/master/icon.png)
```

Comment:
Yes he is. His repo is where the original setup started after 5.1.17 was released. On the next version i’ll can move it to one place.

Jeff


> On May 17, 2016, at 9:10 AM, Razzeee <notifications@github.com> wrote:
> 
> In plugin.video.youtube/README.md <https://github.com/xbmc/repo-plugins/pull/480#discussion_r63518063>:
> 
> > @@ -1,14 +1,8 @@
> > -![](https://raw.githubusercontent.com/bromix/repository.bromix.storage/master/plugin.video.youtube/icon.png)
> > +![](https://raw.githubusercontent.com/Kolifanes/plugin.video.youtube/master/icon.png)
> Is @kolifanes <https://github.com/kolifanes> aware of this? Just to be safe.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/xbmc/repo-plugins/pull/480/files/5ab7db1a24d3b6ff8e2f8a4d594407523dcf890a#r63518063>